src/store/initial-state.js

```diff
@@ -35,6 +35,12 @@ export const createInitialState = () => ({
     miniMap: true,
     modularPipelineFocusMode: null,
   },
+  display: {
+    globalToolbar: true,
+    sidebar: true,
+    miniMap: true,
+    expandAllPipelines: false,
```

Comment:
Hhmm I see what you mean - the main reasoning behind setting up the `expandAllPipelines` field here is to separate / differentiate between the 2 different use case of the `expandAllPipelines` requirement being a react import prop vs bring specified via a flag, with this particular prop aligning to the first use case. 

I've set this state under the `display` object to align with the need to specify the expansion of all modular pipelines in line with the `display.sidebar` prop via the react component import. ( The naming of this prop being `display.expandAllPipelines` rather than `display.modularPipelines` should explain the intent behind this prop in controlling the display intent with expanded pipelines rather than controlling the display of modular-pipelines)

Reply:
I see. So when using Viz as a standalone component, you can do this as you've written it:

```jsx
<KedroViz data={json}  display={{expandAllPipelines: true}} />
```

I'd suggest we don't need to completely align `expandAllPipelines` with how we'd hide the sidebar or mini map. Though related, they act like distinct modifications. So instead of grouping it in with the `display` prop, it could be a first-class, top-level one. In practice, we could do this and it would expand all pipelines:

```jsx
<KedroViz data={json} expandAllPipelines />
```

What do you think?